Automattic/wp-calypso
# Jetpack Plans: add correct set of features to legacy product cards

#### Changes proposed in this Pull Request

* Add the correct set of features to Jetpack Personal, Jetpack Premium, and Jetpack Professional.

#### Testing instructions

* Download this PR.
* Start Calypso Blue with `yarn start`.
* Visit `http://calypso.localhost:3000/plans/[site]?compare_plans=jetpack_personal,jetpack_backup_daily,jetpack_anti_spam` replacing `[site]` with a valid Jetpack site slug.
* Make sure the Jetpack Personal product card shows the following features: `Automated daily backups (off-site)`, and `Comment and form protection`.
* Visit `http://calypso.localhost:3000/plans/[site]?compare_plans=jetpack_premium,jetpack_security_daily` replacing `[site]` with a valid Jetpack site slug.
* Make sure the Jetpack Premium product card shows the following features: `Automated daily backups (off-site)`, `Automated daily scanning`, and `Comment and form protection`.
* Visit `http://calypso.localhost:3000/plans/[site]?compare_plans=jetpack_business,jetpack_security_realtime` replacing `[site]` with a valid Jetpack site slug.
* Make sure the Jetpack Professional product card shows the following features: `Backup (real-time, off-site)`, `Scan (real-time, automated)`, and `Comment and form protection`.

Related to 1200196139286276-as-1200270089536138

#### Demo

**Jetpack Personal**
![image](https://user-images.githubusercontent.com/3418513/116721471-570e7880-a9ab-11eb-9c50-048d82a7ab97.png)

**Jetpack Premium**
![image](https://user-images.githubusercontent.com/3418513/116721491-5bd32c80-a9ab-11eb-8138-5a1b5312c5ca.png)

**Jetpack Professional**
![image](https://user-images.githubusercontent.com/3418513/116721506-61307700-a9ab-11eb-848f-45a6bb0b6b93.png)
